app/src/main/java/com/example/trainingintive/what_to_do_feature/data/repository/ActivityRepositoryImpl.kt

```diff
@@ -1,16 +1,29 @@
 package com.example.trainingintive.what_to_do_feature.data.repository
 
+import androidx.paging.Pager
+import androidx.paging.PagingConfig
+import androidx.paging.rxjava3.flowable
 import com.example.trainingintive.what_to_do_feature.data.network.ActivityApiService
+import com.example.trainingintive.what_to_do_feature.data.pagination.ActivityPagingSource
 import com.example.trainingintive.what_to_do_feature.domain.model.ActivityModel
 import com.example.trainingintive.what_to_do_feature.domain.repository.ActivityRepository
 import io.reactivex.rxjava3.core.Single
+import kotlinx.coroutines.ExperimentalCoroutinesApi
 import javax.inject.Inject
 
 class ActivityRepositoryImpl @Inject constructor(
-    private val activityApiService: ActivityApiService
+    private val activityApiService: ActivityApiService,
+    private val pagingSource: ActivityPagingSource
 ) : ActivityRepository {
 
     override fun getActivity(): Single<ActivityModel> =
         activityApiService.getActivity()
             .map { it.toDomain() }
+
+    @ExperimentalCoroutinesApi
+    override fun getActivities() =
+        Pager(
+            config = PagingConfig(pageSize = 1),
+            pagingSourceFactory = { pagingSource }
+        ).flowable
```

Comment:
You don't need this Flowable because you are not performing any RX operations on this stream. You can obtain LiveData from Pager, it will simplify you ViewModel